frontend/src/hooks/use-user-search/use-user-search.hook.ts

```diff
@@ -0,0 +1,34 @@
+import { SearchValue } from 'common/enums/enums';
+import { useAppDispatch, useSearchParams } from 'hooks/hooks';
+import { chatsActions } from 'store/actions';
+
+type UseCourseSearchResult = {
+  searchParams: URLSearchParams;
+  performSearch: (name: string, value: string) => void;
```

Comment:
```suggestion
  handleSearchPerform: (name: string, value: string) => void;
```
change it also in the `use-course-search` hook